src/enums/name-change-options.ts

```diff
@@ -0,0 +1,5 @@
+export enum NameChangeOptions {
+  CORRECT_NAME = 'correct-name',
+  CORRECT_NAME_TO_NUMBER = 'correct-name-to-number',
+  CORRECT_NEW_NR = 'correct-new-nr'
+}
```

Comment:
This is right from Edit UI. Ultimately this would go in the shared components repo along with the components that use it... future.